src/manage_sql.c

```diff
@@ -21800,19 +21800,13 @@ init_result_get_iterator_severity (iterator_t* iterator, const get_data_t *get,
   column_t *filterable_columns;
   int ret;
   gchar *filter;
-  int autofp, apply_overrides, dynamic_severity;
+  int apply_overrides, dynamic_severity;
   gchar *extra_tables, *extra_where, *owned_clause, *with_clause;
   gchar *with_clauses;
   char *user_id;
 
   assert (report);
 
-  columns[0].select
```

Comment:
This adjusts the columns, but I don't see any adjustment in the columns expected by the caller (report_severity_data).

Reply:
Background: init_result_get_iterator_severity is a special iterator used only for counting the results.  It avoids all the columns used by the general result iterators, so does not benefit from the changes to RESULT_ITERATOR_COLUMNS* and result_iterator_severity etc.